availability-store/src/store.rs

```diff
@@ -41,42 +44,51 @@ mod columns {
 #[derive(Clone)]
 pub struct Store {
 	inner: Arc<dyn KeyValueDB>,
+	candidate_descendents_lock: Arc<Mutex<()>>
 }
 
-fn block_data_key(relay_parent: &Hash, block_data_hash: &Hash) -> Vec<u8> {
-	(relay_parent, block_data_hash, 0i8).encode()
+// data keys
+fn execution_data_key(candidate_hash: &Hash) -> Vec<u8> {
+	(candidate_hash, 0i8).encode()
 }
 
-fn erasure_chunks_key(relay_parent: &Hash, block_data_hash: &Hash) -> Vec<u8> {
-	(relay_parent, block_data_hash, 1i8).encode()
+fn erasure_chunks_key(candidate_hash: &Hash) -> Vec<u8> {
+	(candidate_hash, 1i8).encode()
 }
 
-fn awaited_chunks_key() -> Vec<u8> {
-	"awaited_chunks_key".encode()
+fn candidate_key(candidate_hash: &Hash) -> Vec<u8> {
+	(candidate_hash, 3i8).encode()
 }
 
 fn available_chunks_key(relay_parent: &Hash, erasure_root: &Hash) -> Vec<u8> {
-	(relay_parent, erasure_root, 2i8).encode()
+	(relay_parent, erasure_root, 4i8).encode()
 }
 
-fn block_to_candidate_key(block_data_hash: &Hash) -> Vec<u8> {
-	(block_data_hash, 1i8).encode()
+fn candidates_with_relay_parent_key(relay_block: &Hash) -> Vec<u8> {
+	(relay_block, 5i8).encode()
 }
 
-fn candidate_key(candidate_hash: &Hash) -> Vec<u8> {
-	(candidate_hash, 2i8).encode()
+// meta keys
+fn awaited_chunks_key() -> [u8; 14] {
```

Comment:
This was a minor drive-by cleanup, since it didn't need to be a `Vec`. We will probably have to do further cleanups in the availability-store. Could we say that this is beyond the scope of the PR?

Reply:
Okay :)